blocks/model.py

```diff
@@ -161,15 +162,18 @@ def __init__(self, outputs):
             raise ValueError("top bricks with the same name:"
                              " {}".format(', '.join(repeated_names)))
 
-        brick_param_names = {
-            v: k for k, v in Selector(self.top_bricks).get_params().items()}
-        self.params = []
-        for param in VariableFilter(roles=[PARAMETER])(self.shared_variables):
-            if param in brick_param_names:
-                self.params.append((brick_param_names[param], param))
+        brick_parameter_names = {
+            v: k
+            for k, v in Selector(self.top_bricks).get_parameters().items()}
+        self.parameters = []
```

Comment:
Leaving `parameters` as they currently are makes sense for interface uniformity reasons. Similar methods of the `ComputationGraph` class include `variables`, `auxiliary_variables`, `shared_variables`.

Indeed, the interface which has `parameters`, `get_parameter_dict` and `get_parameter_values` is a bit confusing. But I would like to postpone fixing it to a further PR, because in fact the whole `model.py` file requires major refactoring. I think it makes sense to accomplish the goal of this PR first, that is  to ensure that `params` abbreviation is not used.

Would it be acceptable?